.editorconfig

```diff
@@ -66,8 +66,7 @@ dotnet_style_qualification_for_method = false:warning
 
 dotnet_naming_style.pascal_case.capitalization = pascal_case
 
-dotnet_naming_style.parameters.capitalization = pascal_case
-dotnet_naming_style.parameters.required_prefix = a
+dotnet_naming_style.parameters.capitalization = camel_case
 
 dotnet_naming_style.private_fields.capitalization = pascal_case
 dotnet_naming_style.private_fields.required_prefix = _
```

Comment:
In the standard C# naming convention, private fields are either camelCase or _camelCase.